google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/test/java/com/google/cloud/logging/logback/LoggingAppenderTest.java

```diff
@@ -86,6 +86,7 @@ public void testFlushLevelConfigUpdatesLoggingFlushSeverity() {
                 new ImmutableMap.Builder<String, String>()
                     .put("levelName", "WARN")
                     .put("levelValue", String.valueOf(30000L))
+                    .put("loggerName", LoggingAppenderTest.class.getName())
```

Comment:
Do you think is it better to use `this.getName()` instead of `LoggingAppenderTest.class.getName()`?

Reply:
it should be like `this.getClass().getName()`

Reply:
@rahulKQL PTAL.

Reply:
> it should be like this.getClass().getName()

Are you asking or telling? If telling then LGTM.